### DISCUSSION TITLE
Making spawned vehicles personal.

### DISCUSSION THREAD

**Discussion by mrbxts** [YimMenu/YimMenu#1605] [Q&A] [answered]
Ive been tring to work out a way to make spawned vehicles personal for the last 2 hours.
Ive tried driving them into my garage, LScustoms and nothing has worked.
Ive also tried opening the lscustoms menu and applying tracking to a car but that also doesnt appear to work.
If anyone is aware of a way to do this, please advise.
Thanks

**Comment by CypherMorgan** [comment]
As much I know, Saving spawned vehicles as your personal vehicle comes under the category of Recovery and YimMenu doesn't supports any recovery options. So, I don't really think you'll be able to save those vehicles as your personal vehicles.

**Reply by mrbxts** [reply]
No worries,
Ive managed to find a way to do 1 car at a time by glitching the mod and game fairly easily.
Deluxo garage time!

**Reply by MatusMandzak** [reply, marked as answer]
> No worries, Ive managed to find a way to do 1 car at a time by glitching the mod and game fairly easily. Deluxo garage time!

@mrbxts How did u managed to do so?

**Comment by LintStalker** [comment]
I steal a car off the street that I can drive into my garage, and after the process of entering starts, I bring up Yimmenu and spawn the vehicle I want.  I even did this with an APC

**Comment by Ronu19** [comment]
Yes i tried and here is my trick to make spawn vehicle owner.

session:- invite only

note:- make sure your first list of garage have a space left to get your car

> spawn the vehicle & mod it as you like 

> then hit  "start LS Custom' option inside yimmenu vehicle section

> choose Loss/Theft Prevention

> now buy the "Tracker" & get out of the car

> switch to Franklin

> go online again & call the mechanic

> your car is in your first garage list 🤌

**Reply by NF8S** [reply]
can confirm, this works

**Reply by tyackman** [reply]
Friggin gravedigger. Just spawn a car and drive it into los santos customs. You can put full coverage on it there, then just store it in your garage. Hint (to give it to another player like standheck):
`DECORATOR::DECOR_SET_INT(vehicle, "Player_Vehicle", NETWORK::NETWORK_HASH_FROM_PLAYER_HANDLE(g_player_service->get_selected()->id()));`

**Comment by LintStalker** [comment]
Hmmm, I think it’s easier to steal car of the street, drive it into a
garage and switch during the cut scene.  Doesn’t cost anything.

**Reply by Ronu19** [reply]
Are you able to sell them?

**Comment by LintStalker** [comment]
I haven’t tried.  I do this so I can keep them.  It saves so much money
that I don’t need to make any.
